src/loader/EditorSpineAtlasLoader.ts

```diff
@@ -1,26 +1,20 @@
-import {
-  AssetPromise,
-  Loader,
-  resourceLoader,
-  ResourceManager,
-  LoadItem,
-} from "@galacean/engine";
+import { AssetPromise, Loader, resourceLoader, ResourceManager, LoadItem } from "@galacean/engine";
 import { TextureAtlas } from "@esotericsoftware/spine-core";
 import { createTextureAtlas } from "./LoaderUtils";
 
-
 @resourceLoader("EditorSpineAtlas", ["atlas"])
 class EditorSpineAtlasLoader extends Loader<TextureAtlas> {
-  load(
-    item: LoadItem,
-    resourceManager: ResourceManager
-  ): AssetPromise<TextureAtlas> {
+  load(item: LoadItem, resourceManager: ResourceManager): AssetPromise<TextureAtlas> {
```

Comment:
We want the design to be as general as possible

Reply:
done